corehq/apps/repeaters/models.py

```diff
@@ -348,7 +348,7 @@ def __unicode__(self):
         return "forwarding cases to: %s" % self.url
 
 
-class SOAPRepeaterMixin(object):
+class SOAPRepeaterMixin(Repeater):
```

Comment:
Inheriting from object didn't let me use the `operation` property. Moving this to `Document` or `Repeater` solved it. `Repeater` seemed better option.